data_collection/web/javascript_tracking_client/api.rst

```diff
@@ -0,0 +1,1800 @@
+.. highlight:: js
+.. default-domain:: js
+
+.. _data-collection-javascript-tracking-client-api:
+
+API
+===
+
+The following API allows the user to:
+
+    * track page views
+    * track visits on multiple domains and subdomains
+    * track e-commerce events (successful orders, cart changes, product and
+      category views)
+    * track content impressions
+    * manage custom variables to use them later
+    * track clicked links to external domains and download files
+
+.. contents:: Table of Contents
+
+Installing Tracking code
+------------------------
+
+THIS SECTION WILL BE MOVED TO INSTALLATION DOCUMENT!!!!!!!!!!!!
```

Comment:
Should this line should indeed be published? :)

Reply:
I will remove it.